Sources/App/configure.swift

```diff
@@ -64,9 +64,9 @@ public func configure(_ application: Application) async throws {
 		.schedule(GPXImportingJob())
 		.daily()
 		.at(.midnight)
-	application.queues
-		.schedule(LocationRemovalJob())
-		.everySecond()
+	//application.queues
+	//	.schedule(LocationRemovalJob())
+	//	.everySecond()
```

Comment:
Did you intentionally comment out this job scheduling? Without running `LocationRemovalJob` every second, bus locations will be wildly inaccurate because the location resolution algorithm just assumes that all of the available data in the location history are sufficiently recent.

Reply:
This was commented out for testing the algorithm. This has been resolved.